tests/test_scraper.py

```diff
@@ -1,14 +1,30 @@
-
+import hashlib
+import os
+import re
 import time
 from unittest import TestCase
-import os
-import hashlib
 
+from bs4 import BeautifulSoup
 import httpretty
 import pytest
-from bs4 import BeautifulSoup
 
-from scraper import get_soup, get_image, save_image_file
+
+from scraper import (
+    get_soup, 
+    get_image, 
+    save_image_file, 
+    get_image_name, 
+    get_article_dict, 
+    get_link_list, 
+    change_to_summary_link,
+    get_edition_date, 
+    get_edition_dict, 
+    save_json_file, 
+    get_table_of_contents,
+    get_article_author_date, 
+    get_json_page_dict 
```

Comment:
ten ostatni przecinek może wydawać się dziwny, ale często developerzy zaznaczją cały blok i wykorzystują swoje IDE do posortowania importów alfanumerycznie wówczas `get_json_page_dict` gdzieś bez przecinku i mamy mini błąd, nic poważnego, ale po co tracić na to czas